app/templates/components/pagination-pager.hbs

```diff
@@ -5,7 +5,15 @@
     </li>
 
     {{#each pages key="@index" as |page|}}
-      {{page-item disabled=disabled page=page selected=this.current seperator=seperator urlTemplate=urlTemplate pageSet='pageChanged'}}
+      {{page-item
+        disabled=disabled
+        page=page
+        selected=this.current
+        seperator=seperator
+        urlTemplate=urlTemplate
+        firstPageUrlTemplate=firstPageUrlTemplate
+        pageSet='pageChanged'
+      }}
     {{/each}}
 
     <li class="next {{if isLastDisabled 'disabled'}}">
```

Comment:
I think line 23 here needs something as well since it uses `firstUrl`.

Reply:
Thanks! Good catch! just made the change :+1: